perfectscale/metadata.csv

```diff
@@ -0,0 +1,2 @@
+metric_name,metric_type,interval,unit_name,per_unit_name,description,orientation,integration,short_name,curated_metric
+perfectscale.alert.metric,count,,,,The number of alerts by cluster,0,perfectscale,alert.metric,
```

Comment:
To confirm, you only send in 1 metric?

Reply:
@idan-gazit Can you confirm the integration only sends one metric? The overview says the integration sends in the following: 

"PerfectScale monitors resiliency metrics across your Kubernetes environment and sends these metrics to Datadog through the Integration. When PerfectScale identifies under-provisioned or misconfigured workloads, it promptly pushes an alert to the Datadog dashboard detailing the issue type (such as OOM, CPU-Throttling, Eviction, and more) along with its precise location (cluster, namespace, workload, and container)." 

This integration metadata file says it's only sending the metric that counts the number of alerts by clusters. Thanks!

Reply:
Yes @emarsha94 
We are sending only perfectscale.alert.metric.